### PR TITLE
Add a Mac optimised docker-compose and Bedrock specific docs

### PR DESCRIPTION
**Pull request by MattrCoUk**

### CODE REVIEW

File: bedrock-entry.sh [hunk 1]
```diff
@@ -79,7 +79,7 @@ if [ ! -f "bedrock_server-${VERSION}" ]; then
   done
 
   # ... overwrite all game files, except config files
-  unzip -q ${TMP_ZIP} -x $(ls server.properties whitelist.json permissions.json 2> /dev/null)
+  unzip -q ${TMP_ZIP} -x "$(ls server.properties whitelist.json permissions.json 2> /dev/null)"
```
Comment (itzg) [inline bedrock-entry.sh:82]: I wonder if this change broke the test

Comment (MattrCoUk) [reply bedrock-entry.sh:82]: possibly! my IDE suggested that... shouldn’t have touched :-)

**Comment by MattrCoUk** [comment]
> There's more changes than I expected. I was only interest in the example compose file. I can just pick off that change if you would prefer, rather than going through the PR process.

sure thing!

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      ALLOW_CHEATS: "true"
+      WHITE_LIST: "true"
+    ports:
+      - 19132:19132/udp
+      - 39132:39132
+      - 39132:39132/udp
+      - 39133:39133
+      - 39133:39133/udp
+    sysctls:
+      net.ipv4.ip_local_port_range: 39132 39133
```
Comment (itzg) [inline docker-compose.yml:31]: FYI, when I added this to my local copy the server zip download failed with:
```
bds_1  | DEBU[0000] Using /data to match uid and gid             
bds_1  | DEBU[0000] Resolved UID=0 from match path               
bds_1  | DEBU[0000] Resolved GID=0 from match path               
bds_1  | Failed to extract download URL '' from https://www.minecraft.net/en-us/download/server/bedrock/
bds_1  | 2020/07/15 23:17:12 Failed to get from URLGet https://www.minecraft.net/en-us/download/server/bedrock/: dial tcp: lookup www.minecraft.net on 127.0.0.11:53: read udp 127.0.0.1:39133->127.0.0.11:53: i/o timeout
examples_bds_1 exited with code 2
```

So, I think I'll leave the example compose file as is.

**Pull request closed without merging**